.github/workflows/go.yml

```diff
@@ -18,4 +18,4 @@ jobs:
         with:
           go-version: ${{ matrix.go }}
       - name: Verify benchmarks validate
-        run: go run . --validate
+        run: go test -v -bench . -run bench -benchtime 1ms
```

Comment:
This is not validating the benchmarks (see the job output)

https://github.com/alecthomas/go_serialization_benchmarks/actions/runs/9614218282/job/26518544178?pr=165